src/Polly.Caching.Distributed.nuspec

```diff
@@ -13,6 +13,12 @@
     <tags>Polly Cache Caching Cache-aside</tags>
     <copyright>Copyright © 2017, App vNext</copyright>
     <releaseNotes>
+     1.0.0
```

Comment:
Need to make this 2.0.0 as a version 1.0.x of this package has already been released.